test/ci/243_resources.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": {
+    "major": 1,
+    "minor": 0
```

Comment:
This file appears specific to your CI. Is this correct?

Reply:
Yes

Reply:
I am somewhat against company infrastructure specific code on github. It needs to be identified as such. I would prefer this type of code in a hidden . folder perhaps. Ideally, it shouldn't be here.

Reply:
I suppose we do similar things in our .jenkins folder.

Reply:
Okay. We can hide this.

Reply:
May we can have a script to scan the local computer and build this. Other option is to hook into to rocmsmi via the python interface. Then it can be more generic.

Reply:
Indeed, I wrote a script that calls into `rocminfo` and generates the json. It's a CMake script (independent of the build) to avoid introducing new dependencies, plus making it cross-platform, should ROCm reach Windows one day. README updated accordingly. Code drop incoming.